java/.gitignore

```diff
@@ -0,0 +1,2 @@
+src
```

Comment:
The entire file may not be required at all. Considering that the action is going to be run from a GH action, we don't expect to push back any change.

Reply:
removed, even if I would love to keep `target` in `.gitignore` in case of local development/debug.

Reply:
Okey, we may add it later on if you see it is something useful.